.pre-commit-config.yaml

```diff
@@ -1,11 +1,33 @@
 repos:
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.86.0
+    rev: v1.94.1
     hooks:
       - id: terraform_fmt
       - id: terraform_docs
+        name: terraform_docs on modules
+        args:
+          - --hook-config=--create-file-if-not-exist=true
+          - --args=--hide providers
+        exclude: |
```

Comment:
can we switch to `include infra/modules` pattern instead?

Reply:
probably using `[files](https://pre-commit.com/#top_level-files)`, but I have to give it a try

Reply:
Yes, if we want to specify that the only folder to check is modules, we can use `files: ^infra/modules/.*`, which will automatically exclude everything else